### ISSUE TITLE
Highlight current windw / tab

### ISSUE THREAD

**Issue by x3rAx** [stefanXO/Tab-Manager-Plus#52]
Hi, thanks for this great extension!

I would love to see the active window and tab (also the one of other windows) highlighted. 

Maybe by a fat border or simply an indicator:

![Proposal](https://i.imgur.com/WtnxEEa.png)
(showing off my best paint skills here)

I think I like the third version the best. I would not put the window indicator at the bottom, as it might not be visible. With many tabs open and a small extension window, the indicator might be out of the visible area. With the background version, the active window would be visible regardless of its size and I personally find the indicator for the tab much more elegant.

One thing to add: I know that the active window is always the top-left window but I tend to forget this and begin to think about which window is the active one. A simple highlighting would IMHO make a much better user experience.

**Comment by mastef** [comment]
Hi @x3rAx , there should be a purple border currently with a glow - doesn't it show up for you?

![image](https://user-images.githubusercontent.com/762297/73131588-5072b200-4040-11ea-99ec-4bf0110b738f.png)

**Comment by x3rAx** [comment]
I can confirm that it works in Chromium:

![Imgur](https://i.imgur.com/CVlZTYt.png)

But in Opera the `highlighted` class seems to be missing:

![Imgur](https://i.imgur.com/h3Y5r4M.png)

I'm using Opera (latest version 66.0.3515.44) and the highlighting is missing on both Linux and Windows.

Now I know you don't support Opera, but it is chromium based and extensions should actually just work™. I never had noticed anything that didn't work in Opera extensions and it should be pretty easy to release for opera (as far as I know, the process is the same as for Chrome but you upload it to https://addons.opera.com/ instead).

So I would rephrase my issue:

# Please support Opera

Thanks for considering <3
